repos/system_upgrade/common/actors/checkconsumedassets/actor.py

```diff
@@ -0,0 +1,18 @@
+from leapp.actors import Actor
+from leapp.libraries.actor import check_consumed_assets
+from leapp.models import ConsumedDataAsset, LeappRepositoryInfo, Report
+from leapp.tags import ChecksPhaseTag, IPUWorkflowTag
+
+
+class CheckConsumedAssets(Actor):
+    """
+    Check that leapp used correct data assets.
```

Comment:
```suggestion
    Check whether Leapp is using correct data assets.
```

Reply:
Resolved with the latest push as suggested.